src/coq_rules.ml

```diff
@@ -90,11 +120,42 @@ let create_ccoq sctx ~dir =
   ; coqpp  = rr "coqpp"
   }
 
+(* compute include flags and mlpack rules *)
+let setup_ml_deps ~scope ~stdlib_dir libs =
+
+  (* coqdep expects an mlpack file next to the sources otherwise it
+   * will omit the cmxs deps *)
+  let ml_pack_files lib =
+    let plugins = Mode.Dict.get (Lib.plugins lib) Mode.Native in
```

Comment:
Why native only? What if it's a bytecode only lib?

Reply:
For now I am only supporting the native version of coq, I do agree that it would be cool to support both modes but I am leaving that to "phase 2" as it may require a few tweaks.